core/common/src/main/java/alluxio/underfs/UnderFileSystemConfiguration.java

```diff
@@ -11,13 +11,18 @@
 
 package alluxio.underfs;
 
+import alluxio.Constants;
 import alluxio.annotation.PublicApi;
 import alluxio.conf.AlluxioConfiguration;
 import alluxio.conf.AlluxioProperties;
 import alluxio.conf.ConfigurationValueOptions;
 import alluxio.conf.InstancedConfiguration;
+import alluxio.conf.PropertyKey;
 import alluxio.conf.Source;
 
+import org.apache.hadoop.conf.Configuration;
```

Comment:
it can be tricky to introduce dependency of `org.apache.hadoop` in common.
possible to move it to master module?

Reply:
Ok, move the logic into master module.